README.md

```diff
@@ -40,19 +40,19 @@
 
 We love the default GitHub profiles and we want to enhance them:
 
-* The GitHub profiles aren't clearly showing all repos you have contributed to since you joined
+* The GitHub profiles clearly aren't showing all the repos you have contributed to since you joined
```

Comment:
1. aren't clearly showing them = showing them, but not in a clear way
2. clearly aren't showing them = not showing them at all

We do mean 1 :)

"the" -> OK